cypress/integration/cucumber/step_definitions/refapp-3.x/06-patient-search/patient-search.js

```diff
@@ -9,14 +24,19 @@ Given('the user login to the Registration Desk', () => {
     cy.visit('home');
 })
 
-When('the user search for {string}', patientName => {
+When('the user search for {string}', () => {
```

Comment:
We don't need this parameter.
```suggestion
When('the user searches for a patient by name', () => {
```